variants/vueapp/src/components/FilterApp.vue

```diff
@@ -240,6 +240,12 @@ watch(
         :prio-enabled="
           variantQueryStore.previousQueryDetails.query_settings.prio_enabled
         "
+        :face-enabled="
+          variantQueryStore.previousQueryDetails.query_settings.face_enabled
+        "
+        :prio-face="
```

Comment:
What exactly does this value do?

Reply:
It stores the gestalt scores received from the GestaltMatcher API.